src/commons/yaml_parser.py

```diff
@@ -119,9 +119,12 @@ def test_parser(yaml_file, number_of_nodes):
         if "object_size" not in data:
             raise AssertionError(
                 f"Object size is compulsory, which is missing in workload {yaml_file}")
-        convert_object_part_size_to_bytes(data)
-        convert_range_read_to_bytes(data)
-        convert_min_runtime_to_time_delta(test, delta_list, data)
+        if "total_samples" in data and isinstance(data["object_size"], dict):
+            convert_object_size_to_bytes_samples(data)
```

Comment:
This input condition needs to be documented in docs/YAML_documents/yaml_structure.md file

Reply:
Updated the doc with parameter "total_samples".